test/it/fileutils/ln_gnu_test.sh

```
@@ -0,0 +1,27 @@
+Setup() {
+    export TEST_DIR=${MIMIXBOX_IT_ROOT}/ln_gnu
+    mkdir -p ${TEST_DIR}/a
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)